src/game/DBScripts/ScriptMgr.cpp

```diff
@@ -1541,6 +1604,30 @@ bool ScriptAction::ExecuteDbscriptCommand(WorldObject* pSource, WorldObject* pTa
             if (LogIfNotUnit(pSource))
                 break;
 
+            Creature* creature = static_cast<Creature*>(pSource);
+
+            if (m_script->textId[0])
+            {
+                if (m_script->textId[0] == 1)
+                {
+                    auto respPos = creature->GetRespawnPosition();
```

Comment:
Shauren kindly explained to me not to use auto like this. I know what the type is, but anyone else reviewing wont. So in these cases dont use auto and put Position there.

Reply:
Lazyness spotted you are correct